### ISSUE TITLE
remove instrument response fails after calibration.

### ISSUE THREAD

**Issue by kkappler** [kujaku11/mth5#252]
This is new.  I added a fix for [earthscope-mt-course issue 10](https://github.com/simpeg-research/earthscope-mt-course/issues/10) and then, re-ran the part of the notebook that doesn't build the mth5, but just does some calibration and plotting ... 

I found that once we have calibrated the channel, upon closing and re-opening the file, if we go to cell 9
```
# open file already created
mth5_object = MTH5()
mth5_object = mth5_object.open_mth5("4P_WYYS2_MTF20_WYYS3_MTC20.h5")
```

and then execute the notebook from there forward, cell 18
```
ex.remove_instrument_response(plot=True)
```
fails with 

> File "/home/kkappler/software/irismt/earthscope-mt-course/notebooks/mth5/executables/05_make_mth5_from_iris_v0.2.0.py", line 116, in <module>
    ex.remove_instrument_response(plot=True)
  File "/home/kkappler/software/irismt/mth5/mth5/timeseries/channel_ts.py", line 1255, in remove_instrument_response
    calibrated_ts.ts = remover.remove_instrument_response(
  File "/home/kkappler/software/irismt/mth5/mth5/timeseries/ts_filters.py", line 535, in remove_instrument_response
    f = np.fft.rfftfreq(ts.size, d=self.sample_interval)
  File "/home/kkappler/anaconda3/envs/aurora/lib/python3.9/site-packages/numpy/fft/helper.py", line 218, in rfftfreq
    val = 1.0/(n*d)

Somehow the metdata for the channel in the mth5 has been altered when we calibrated ...

**Comment by kkappler** [comment]
How to debug this:
- run 05_make_mth5_from_iris_v0.2.0.ipynb up to cell 8, this creates a fresh copy of 4P_WYYS2_MTF20_WYYS3_MTC20.h5
- copy that file to a scratch folder so that we can try to identify what file property is changing when we calibrate

**Comment by kkappler** [comment]
and the issue is not a bug, but the fact that the notebook is getting the channel from iloc[-6] in the channel_summary dataframe.  Once the TFs are added, the channel summary dataframe, gets new rows, and iloc[-6] points at a TF, not a channel.

@kujaku11 is it the intention that the channel_summary show the TFs as well?  Seems reasonable enough but maybe we can add a kwarg that would make the df be timeseries' only?

**Comment by kujaku11** [comment]
@kkappler good catch. The way the tfs are setup is that it pulls metadata from the channel, runs, etc. If there are not channels when the tf is saved it creates them.  Therefore, the channel summary will find them.  that was the intention cause if you have a time series filled mth5, the metadata should be pulled from them. I don't think there is a way to decipher between channels that are linked to a transfer function or not. Unless we assume dataless channels are attached to tfs.   I think it is update to the user to query the channel summary, we are simply providing the information.  Does that sound reasonable?

**Comment by kujaku11** [comment]
closed with PR #253
